runtime/composable/src/lib.rs

```diff
@@ -749,6 +749,7 @@ pub struct BaseCallFilter;
 
 impl Contains<Call> for BaseCallFilter {
 	fn contains(call: &Call) -> bool {
+		// TODO: not up to date as whole composable setup
```

Comment:
What do you mean by this

Reply:
i do not updtae composable runtime. and not updatwd democracy and xcmp here too. and others do not update it too. so likely need at some point to recheck whole runtime and update.